conf.yaml

```
@@ -527,7 +527,7 @@ contents:
                     path:   .doc/
               - title:      .NET Clients
                 prefix:     net-api
-                current:    8.0
+                current:    8.1
                 branches:   [ {main: master}, 8.7, 8.6, 8.5, 8.4, 8.3, 8.2, 8.1, 8.0, 7.17, 7.16, 6.x, 5.x, 2.x, 1.x ]
                 live:       [ main, 8.0, 7.17, 7.16, 6.x, 5.x, 2.x, 1.x ]
```

Please add 8.1 to the live branches, too.

```suggestion
                live:       [ main, 8.1, 8.0, 7.17, 7.16, 6.x, 5.x, 2.x, 1.x ]
```